ui/admin/app/components/form/auth-method/oidc/index.hbs

```diff
@@ -406,45 +404,43 @@
 
   </Form::Field::ListWrapper>
 
-  <form.input
-    @name='max_age'
-    @type='number'
+  <Hds::Form::TextInput::Field
+    name='max_age'
```

Comment:
We should not remove the type here, too. Can we add the number type back in?

Reply:
@DhariniJeeva `type="number"` is not supported in `Hds::TextInput::Field`.  The [documentation](https://helios.hashicorp.design/components/form/text-input?tab=code#formtextinputfield) states it causes accessibility issues.  Even if I try and use `type="number"` here it still comes back as a `type="text"`.